### PR TITLE
feature(common):add order confirmation

### PR DESCRIPTION
**Pull request by MailyCristi**
adding the confirmation page

### CODE REVIEW

**Comment by olmedo90** [comment]
image

It is not according to Figma. correct the following: (color of the components, color and size of the texts)

**Comment by olmedo90** [comment]
image

It is according to Figma. Continue with the same interest.👍

File: src/ui/components/ConfirmacionDeCompra/ConfirmacionDeCompra.jsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  createDatos(1879605573994,'Standard delivery','Vitathemes@gmail.com','Kristian holst 34 old street W1F 7NU london United Kingdom','*************7865',448749790988,'gaegagag'),
+];
+
+const ConfirmacionDeCompra = () => {
+    const classes=useStyles();
+  return (
+    <>
+        <Box className={classes.base}>  
+                <Box className={classes.confirmacion}>
+                        <svg className={classes.icon} width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Ditmar) [inline src/ui/components/ConfirmacionDeCompra/ConfirmacionDeCompra.jsx]: use the icon component

File: src/ui/components/ConfirmacionDeCompra/ConfirmacionDeCompra.jsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+const rows = [
+  createData(411,'lorem insup',60),
+  createData(411,'lorem insup',100),
+  createData(411,'lorem insup',80),
+];
+const rowss = [
+  createDatos(1879605573994,'Standard delivery','Vitathemes@gmail.com','Kristian holst 34 old street W1F 7NU london United Kingdom','*************7865',448749790988,'gaegagag'),
+];
+
+const ConfirmacionDeCompra = () => {
```
Comment (Ditmar) [inline src/ui/components/ConfirmacionDeCompra/ConfirmacionDeCompra.jsx]: use english

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments below

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**